src/components/FoodOrderComponents/UsersOrderStatus.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <v-content v-if="$apollo.loading" class="loading-warpper">
+    <v-progress-circular :size="40" color="primary"
+      indeterminate/>
+  </v-content>
+  <v-content v-else>
+    <v-container class="container">
+        <v-checkbox
+        class="checkbox"
+        :key="user.id" v-for="user in users"
+        :label="user.name"
+        :disabled="!user.isLoggedInUser"
+        v-model="user.isOrderingToday"
+        @change="updateOrdersAmount(user.isOrderingToday)"/>
+    </v-container>
+    <v-divider />
+    <v-footer color="transparent" class="footer">
+          <h5>{{`מספר מזמינים: ${this.ordersAmount}`}}</h5>
+          <v-btn @click="onNextButtonClicked">
+          המשך
+          </v-btn>
+          <h5 class='error-message'>{{this.errorMessage}}</h5>
+    </v-footer>
+  </v-content>
+</template>
+
+<script lang="ts">
+import { isToday } from 'date-fns';
+import { getModule } from 'vuex-module-decorators';
+import {
+  Component, Vue, Prop,
+} from 'vue-property-decorator';
+
+import User from '@/models/User';
+import StoreModule from '@/store/storeModule';
+import { AllUsers } from '@/db-service/Users/queries';
+import { updateLastFoodOrder } from '@/db-service/Users/mutations';
+import { dbErrorMessage } from '@/utils/errorMessages';
+
+interface UserOrderStatus {
+  name: string;
+  isLoggedInUser: boolean;
+  isOrderingToday: boolean;
+}
+
+@Component({
+  apollo: {
+    allUsers: {
+      query: AllUsers,
+      fetchPolicy: 'network-only',
+    },
+  },
+})
+export default class UsersOrderStatus extends Vue {
+  @Prop({ type: Function })
+  passToNextStep!: Function;
```

Comment:
Why not using an emit? instead using a prop you can trigger a custom event, look in categories bar in the calendar

Reply:
fixed at 12bb8ad